contracts/truefi/TruPriceChainlinkOracle.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {ITruPriceOracle} from "./interface/ITruPriceOracle.sol";
+import {IUniswapPair} from "./interface/IUniswapPair.sol";
+
+interface IChainLink {
+    function latestAnswer() external view returns (int256);
+}
+
+contract TruPriceChainLinkOracle is ITruPriceOracle {
+    using SafeMath for uint256;
+
+    IChainLink public chainlinkOracle;
+
+    constructor(IChainLink _chainlinkOracle) public {
+        chainlinkOracle = _chainlinkOracle;
+    }
+
+    /**
+     * @dev converts from USD with 18 decimals to TRU with 8 decimals
+     */
+    function usdToTru(uint256 amount) external override view returns (uint256) {
+        return amount.div(safeUint(chainlinkOracle.latestAnswer())).div(100);
```

Comment:
I see that `latestAnswer()` is deprecated in favor of `latestRoundData()`, though it also doesn't look like the distinction is important.

Reply:
not sure if this matters, so marking as resolved